lib/src/Uncertainty/Algorithm/MetaModel/MetaModelAlgorithm.cxx

```diff
@@ -43,14 +44,39 @@ MetaModelAlgorithm::MetaModelAlgorithm()
   // Nothing to do
 }
 
+/* Constructor with parameters */
+MetaModelAlgorithm::MetaModelAlgorithm(const Sample & inputSample,
+                                       const Sample & outputSample)
+  : MetaModelAlgorithm(inputSample, outputSample, Dirac(Point(inputSample.getDimension(), 0.0)))
```

Comment:
Why did you choose a Dirac distribution here? A UserDefined one would have been more adapted from a statistical point of view. Is it related to the cost of constructing such a distribution?

Reply:
I dont think its actually used, but UserDefined is ok too